tests/UnitTests/CoreGraphics.drawing/DrawingTest.cpp

```diff
@@ -15,9 +15,9 @@
 //******************************************************************************
 
 #include "DrawingTest.h"
-
-#include <CoreFoundation/CoreFoundation.h>
-#include <ImageIO/ImageIO.h>
+#include "DrawingTestConfig.h"
```

Comment:
Not in a C++ file.

---
In reply to: [89410332](https://github.com/Microsoft/WinObjC/pull/1438#discussion_r89410332) [](ancestors = 89410332)